# service to set parameter of a media (camera mostly)

string media_name

string param_name

float32 value

---

uint8 SUCCESS = 1
uint8 FAIL = 0
# boolean, 1 success, 0 fail.
uint8 success
